BackendWebApp/src/main/java/uk/ac/ebi/biostd/webapp/application/security/rest/GroupsMapper.java

```diff
@@ -17,14 +20,20 @@ UserGroup toGroup(UserGroupDto userGroupDto) {
         group.setOwner(userRepository.getOne(userGroupDto.getOwnerId()));
         group.setName(userGroupDto.getName());
         group.setDescription(userGroupDto.getDescription());
+        group.setId(userGroupDto.getGroupId());
         return group;
     }
 
     UserGroupDto toDto(UserGroup group) {
         UserGroupDto groupDto = new UserGroupDto();
+        groupDto.setGroupId(group.getId());
         groupDto.setName(group.getName());
         groupDto.setDescription(group.getDescription());
         groupDto.setOwnerId(group.getOwner().getId());
         return groupDto;
     }
+
+    List<UserGroupDto> toDtoList(List<UserGroup> usersGroups) {
```

Comment:
I've seen in other mappers we use `map` as the method name. Do we want to use it as a standard?